src/libpriv/rpmostree-passwd-util.c

```diff
@@ -850,44 +767,35 @@ _rpmostree_gfile2stdio (GFile         *source,
                         GCancellable  *cancellable,
                         GError       **error)
 {
-  gboolean ret = FALSE;
   gsize len;
-  FILE *src_stream = NULL;
+
+  *ret_src_stream = NULL;
```

Comment:
It seems odd to be modifying output vars here. Looking at the two callers of this function, I see that `src_stream` is already initialized to `NULL` so they weren't relying on this. Let's just nix this to be more consistent with the new style?